bot/cogs/tickets.py

```diff
@@ -0,0 +1,405 @@
+    @commands.hybrid_command(name="ticket-info", aliases=["tinfo"])
+    async def ticket_info(self, ctx: RoboContext) -> None:
+        """Provides information about the current ticket"""
+        ticket = await get_cached_thread(self.bot, ctx.author.id, self.pool)
+        partial_ticket = await get_partial_ticket(self.bot, ctx.author.id, self.pool)
+        if ticket is None or partial_ticket.id is None:
+            await ctx.send(
+                "You have no active tickets. Please send a message to Rodhaj to get started"
+            )
+            return
+
+        formatted_tags = ", ".join(
+            tag.name for tag in ticket.thread.applied_tags
+        ).rstrip(",")
+        ticket_owner = self.bot.get_user(partial_ticket.owner_id) or (
+            await self.bot.fetch_user(partial_ticket.owner_id)
+        )
+        embed = Embed()
+        embed.title = f"\U0001f3ab {ticket.thread.name}"
+        embed.description = formatted_tags
+        embed.add_field(name="Is Active", value=ticket is not None, inline=False)
+        embed.add_field(
+            name="Is Closed",
+            value=ticket.thread.archived and ticket.thread.locked,
+            inline=False,
+        )
+        embed.add_field(name="Ticket Owner", value=ticket_owner.mention, inline=False)
+        embed.add_field(
+            name="Associated Guild", value=ticket.source_guild.name, inline=False
+        )
+        embed.add_field(name="Created At", value=format_dt(ticket.thread.created_at), inline=False)  # type: ignore
+        await ctx.send(embed=embed)
+
+    # As the guild has an entry in the cache,
+    # we need to invalidate it if a guild goes
+    @commands.Cog.listener()
+    async def on_guild_remove(self, guild: discord.Guild) -> None:
+        dispatcher = GuildWebhookDispatcher(self.bot, guild.id)
+        dispatcher.get_config.cache_invalidate()
+
+    @commands.Cog.listener()
+    async def on_ticket_create(
+        self,
+        guild: discord.Guild,
+        user: Union[discord.User, discord.Member],
+        ticket: discord.channel.ThreadWithMessage,
+        init_message: str,
+    ) -> None:
+        webhook = await self.obtain_webhook(guild.id)
+
+        if webhook is not None:
+            embed = LoggingEmbed(title="\U0001f3ab New Ticket")
+            embed.description = init_message
+            embed.add_field(name="Owner", value=user.mention)
+            embed.add_field(name="Link", value=ticket.thread.mention)
+            await webhook.send(embed=embed)
+
+    @commands.Cog.listener()
+    async def on_ticket_close(
+        self,
+        guild: discord.Guild,
+        user: Union[discord.User, discord.Member],
+        ticket: discord.Thread,
+        author: Optional[Union[discord.User, discord.Member]] = None,
+    ) -> None:
+        webhook = await self.obtain_webhook(guild.id)
+
+        if webhook is not None:
+            embed = LoggingEmbed(
+                title="\U0001f512 Ticket Closed",
```

Comment:
```py
title="\U0001f512 Ticket Closed", # U+1F3AB Ticket
```

Reply:
Fixed in commit adee012304df96d8a492c485c54dedcdd8a879bb